src/common/ast-path.js

```diff
@@ -20,6 +21,30 @@ class AstPath {
     this.stack = [value];
   }
 
+  get name() {
```

Comment:
I know we used "getName" before, but do you think `key` is a better choice?

Reply:
I like `key` better too. Let's rename it.